DataFormats/MuonReco/src/Muon.cc

```diff
@@ -74,6 +100,14 @@ int Muon::numberOfMatches( ArbitrationType type ) const
          matches++;
          continue;
       }
+      if(type == ME0SegmentAndTrackArbitration) {
+	if (chamberMatch->id.subdetId() == MuonSubdetId::ME0) matches += chamberMatch->segmentMatches.size();
+	continue;
+      }
+      if(type == GEMSegmentAndTrackArbitration) {
+	if (chamberMatch->id.subdetId() == MuonSubdetId::GEM) matches += chamberMatch->segmentMatches.size();
```

Comment:
This is just a placeholder until the me0 and gem performance has been studied. For now we just need number of ME0 or GEM matches.